### PR TITLE
using global-object to get global.Element so this code can be safely run inside of non-browser environments.

### PR DESCRIPTION
**Pull request by antialias**

### CODE REVIEW

**Comment by antialias** [comment]
bump @ockham

**Comment by antialias** [comment]
bump @ockham

**Comment by antialias** [comment]
bump @ockham

**Comment by antialias** [comment]
bump @tj @TooTallNate

File: index.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ try {
  * Element prototype.
  */
 
-var proto = Element.prototype;
+var Element = require('global-object').Element;
```
Comment (twhid) [inline index.js]: Do you need to add another dep?

It appears to me that just...
```js
 var proto = typeof Element !== 'undefined' ? Element.prototype : {};
```

...would achieve what is desired.

I understand that `global-object` is tiny and its entire purpose is as you're using it, but a more surgical PR might be more welcome.

Comment (antialias) [inline index.js]: Seemed more immediately comprehensible to use `global-object`. Happy to go with whatever the maintainers like in order to get this merged though.

**Comment by gasi** [comment]
👍

File: index.js (hunk 1 above)
Comment (LinusU) [inline index.js]: I think it's better to use another dep then to copy and paste the code into this library...

**Comment by LinusU** [comment]
ping @ockham, would love to see this merged :)

**Comment by GeorgeTaveras1231** [comment]
Any update on this? would be great to get this fixed

**Comment by ockham** [comment]
Sorry for not noticing this any sooner 😳

**Pull request merged**